.github/ISSUE_TEMPLATE/question.md

```diff
@@ -8,3 +8,4 @@ assignees: ''
 ---
 
 When asking a question please also include where you looked for an answer (so we can update the documentation if needed).
+General questions regarding SRT are encouraged to be asked in the SRT Alliance slack channel: [join](https://slackin-srtalliance.azurewebsites.net/).
```

Comment:
You are encouraged to ask general questions regarding SRT in the [SRT Alliance Slack channel](https://slackin-srtalliance.azurewebsites.net/).